src/command_line_options.cpp

```diff
@@ -98,23 +102,25 @@ void Command_line_options::print_usage(std::ostream& ostr) const
 {
    ostr << "Usage: " << program << " [options]\n"
            "Options:\n"
-           "  --slha-input-file=<filename>      SLHA input file (default: )\n"
-           "                                    If <filename> is - then the"
-                                                " SLHA input\n"
-           "                                    is read from stdin.\n"
-           "  --slha-output-file=<filename>     SLHA output file (default: -)\n"
-           "                                    If <filename> is the empty string, then\n"
-           "                                    no SLHA output is written.\n"
-           "                                    If <filename> is - then the output is\n"
-           "                                    printed to stdout.\n"
-           "  --spectrum-output-file=<filename> file to write spectrum to\n"
-           "  --database-output-file=<filename> SQLite database file to write\n"
-           "                                    parameter point to\n"
-           "  --rgflow-output-file=<filename>   file to write rgflow to\n"
-           "  --build-info                      print build information\n"
-           "  --model-info                      print model information\n"
-           "  --help,-h                         print this help message\n"
-           "  --version,-v                      print program version"
+           "  --slha-input-file=<filename>       SLHA input file (default: )\n"
+           "                                     If <filename> is - then the"
+                                                 " SLHA input\n"
+           "                                     is read from stdin.\n"
+           "  --slha-output-file=<filename>      SLHA output file (default: -)\n"
+           "                                     If <filename> is the empty string, then\n"
+           "                                     no SLHA output is written.\n"
+           "                                     If <filename> is - then the output is\n"
+           "                                     printed to stdout.\n"
+           "  --spectrum-output-file=<filename>  file to write spectrum to\n"
+           "  --database-output-file=<filename>  SQLite database file to write\n"
+           "                                     parameter point to\n"
+           "  --higgsbounds-dataset=<directory>  Location of HiggsBounds dataset\n"
```

Comment:
Should description always start with a small letter? It looks like it but the `If <filename>` confused me.

Reply:
Yes, I think it would be best to start with a capital letter.

Reply:
Done in 7358cbc12d2a3369718babc5a7b58a29606ba531.